PocketKit/Sources/PocketKit/MyList/ArchivedItemsList/ArchivedItemsListViewModel.swift

```diff
@@ -103,20 +117,43 @@ class ArchivedItemsListViewModel: ItemsListViewModel {
         snapshot.appendItems(itemCellIDs, toSection: .items)
         return snapshot
     }
-
-    func toggleFavorite(_ cell: ItemsListCell<String>) {
-
-    }
-
-    func archive(_ cell: ItemsListCell<String>) {
-
+    
+    private func blankSnapshot() -> Snapshot {
+        var snapshot = Snapshot()
+        snapshot.appendSections(ItemsListSection.allCases)
+        
+        snapshot.appendItems(
+            ItemsListFilter.allCases.map { ItemsListCell<ItemIdentifier>.filterButton($0) },
+            toSection: .filters
+        )
+        
+        return snapshot
     }
+    
+    private func apply(filter: ItemsListFilter, from cell: ItemsListCell<ItemIdentifier>) {
+        if selectedFilters.contains(filter) {
```

Comment:
So I like this approach. In `SavedItemsListViewModel` we actually have `selectedFilters` publish its changes and then we subscribe to those changes and do the actual fetch within that subscription.
It would be nice to take a consistent approach in both view models, would you mind updating `SavedItemsListViewModel` to follow this simpler implementation?

I could also see doing something like this:
```
var selectedFilters: [ItemsListFilter] {
  didSet { applyFilters() }
}

func applyFilters() {
  // apply the blank snapshot
  // fetch()
}
```